addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -367,6 +368,15 @@ public Map<String, String> getAlertTags() {
         return ALERT_TAGS;
     }
 
+    private Map<String, String> getNeededAlertTags(TestType type) {
+        Map<String, String> alertTags = new HashMap<>();
+        alertTags.putAll(getAlertTags());
```

Comment:
This should only be done if needed, there's no reason to create a copy if we are not removing the tag.